ui/raidboss/popup-text.ts

```diff
@@ -317,7 +317,8 @@ class TriggerOutputProxy {
   getReplacement(
       // Can't use optional modifier for this arg since the others aren't optional
       template: { [lang: string]: unknown } | string | undefined,
-      params: TriggerParams,
+      // User trigger may pass unknown value as parameters
+      params: { [key: string]: unknown },
```

Comment:
sorry, this force pushed is because I mis-add some irrelevant changes.

I'm not very good at this pr reviewing, what if I need to resolve conflicts? Should I merge main into pr branch? Sounds like I shouldn't rebase and force push.

Reply:
I think when somebody makes changes to a PR, it's easier for me if there's just an extra commit (with no rebase or merge).  Then I can click on the commit (e.g. a4aecab above) and see what has changed there.  If it's a force push then there's no way to tell what was changed.  If it's just a "I uploaded this and immediately saw a typo and force pushed to fix it", maybe I just started reviewing at the wrong time.  :)

I think usually I would say don't rebase onto main or merge main unless there's some change that you absolutely need from the main branch to make the PR work or a conflict.  I think if there is, merging is slightly better (because it keeps all the old commits) but force pushing is also ok, so long as it's not that often and you keep all the other commits you've made intact.  But ideally force pushing should be really infrequent.